build-system/babel-plugins/babel-plugin-transform-stringish-literals/index.js

```diff
@@ -34,79 +30,78 @@ module.exports = function ({types: t}) {
     }
   }
 
+  function joinTemplateLiterals(left, right) {
+    // First merge the first member of the right quasi into the last quasi on the left.
+    const fromQuasi = right.get('quasis.0');
+    const toQuasi = left.get(`quasis.${left.node.quasis.length - 1}`);
+    toQuasi.node.value.raw += fromQuasi.node.value.raw;
+    toQuasi.node.value.cooked += fromQuasi.node.value.cooked;
+
+    // Can safely remove the merged quasi.
+    fromQuasi.remove();
```

Comment:
We're removing the entire right template literal below.

Reply:
This removes the quasi from the items on the right before merging. Otherwise we need to filter it during the push phase below.

Reply:
I'll add a comment.

Reply:
There's a risk this will throw in Babel 8 (which has stricter AST checking). Maybe use `slice` on the quasi push?